package.json

```diff
@@ -80,6 +80,7 @@
     "@apollo/client": "3.8.1",
     "@azure/msal-angular": "3.0.16",
     "@azure/msal-browser": "3.13.0",
+    "@graphql-codegen/fragment-matcher": "5.0.2",
```

Comment:
question: does this need to be under "dependencies" and not "devDependencies"?

Reply:
4c4fd025c49fc48d19f614bd4db40b6516e7d15b